APE0.rst

```diff
@@ -0,0 +1,360 @@
+Project. The Astropy Project broadly follows the philosophy of do-ocracy:
+project participants who gain experience and knowledge through sustained
+community contributions and have demonstrated adherence to core principles of
+inclusion and consensus-building will be entrusted with increasing levels of
+authority. This peaks with the Coordination Committee, which has relatively
+broad authority but seeks to exercise it as rarely as possible by working
+towards consensus. This is because at its core, the Project is both by and for
+its community, and therefore the community is the final source of authority and
+direction.
+
+Detailed Description
+====================
+The Governance Charter is described in this section. The initial version is the
+outcome of several months of discussion by the Astropy Governance Working Group,
+which aimed to both codify existing informal governance rules as well as update
+them with new ideas that are consistent with the philosophy of and vision for
+the Astropy project.
+
+Ultimate Authority: The Astropy Community
+-----------------------------------------
+The Astropy Project derives its authority from the whole community - developers,
+users, and even potential participants in the broader astronomical and
+scientific software community. All have a vested interest in the success of the
+Project, so all governance in the end belongs to this community in its entirety.
+The governance Charter described below provides levels of delegation of that
+authority, particularly to the Voting Members and the Coordination Committee.
+
+The Coordination Committee
+--------------------------
+
+Composition
+^^^^^^^^^^^
+The Coordination Committee is a committee that is composed of four members. To
+represent a broad set of interests, the Coordination Committee should have no
+more than two members from a particular employer. The Ombudsperson (described in
+Section 4) and the Coordination Committee will work to maintain this composition
+following the processes laid out in this Charter.
+
+Role
+^^^^^^^
+The Coordination Committee’s role is to facilitate consensus in the Project and
+coordinate Project-spanning efforts. As such, the Coordination Committee shall
+work to:
+
+* Maintain the quality and stability of the Astropy ecosystem of packages for
+  the community,
+* Facilitate the development of resources that improve accessibility and
+  understanding of the Astropy ecosystem,
+* Make contributing as accessible, inclusive, and sustainable as possible,
+* Oversee the financial and legal status of the project,
+* Lead and oversee the decision-making process for APEs and other community
+  decisions,
+* Seek consensus among the Astropy community, particularly before acting in a
+  formal capacity,
+* Act as the decision-making body where all other methods for achieving
+  consensus have failed,
+* Operate openly and provide transparency and opportunity for community input
+  into its activities.
+
+Powers
```

Comment:
How about "role" or "responsibility" instead of "power"?

Reply:
Hah, we now have three threads with various options for "power" alternatives... I think I may like "responsibilities" thus far, but see the thread in https://github.com/astropy/astropy-APEs/pull/61#discussion_r501251734

Reply:
(see thread below on this)